config/vxlan.py

```diff
@@ -3,8 +3,10 @@
 
 from jsonpatch import JsonPatchConflict
 from .validated_config_db_connector import ValidatedConfigDBConnector
+from swsscommon.swsscommon import validate_interface_name_length, iface_name_max_length
 
 ADHOC_VALIDATION = True
+IFNAMSIZ = 16
```

Comment:
>16 [](http://example.com/codeflow?start=11&length=2)

Is it possible to get the value programmatically? In future, Linux may support longer ifName.

Reply:
or centralize it inside sonic repos

Reply:
Thanks.
I removed it, this actually wasn't used (I forgot to delete it) - I was using the value imported from sonic-swss-common: line #29